crates/dips/src/lib.rs

```diff
@@ -754,7 +783,17 @@ mod test {
             subgraphDeploymentId: voucher_ctx.deployment_id.clone(),
         };
 
-        let low_price_voucher = voucher_ctx.test_voucher(metadata);
+        let low_entity_price_voucher = voucher_ctx.test_voucher(metadata);
+
+        let metadata = SubgraphIndexingVoucherMetadata {
+            basePricePerEpoch: U256::from(10_u64),
+            pricePerEntity: U256::from(10000_u64),
```

Comment:
Idem

Reply:
Not really, these are hardcoded values for test which require U256